src/components/EventManagement/AddNewEvents.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import { eventPostData } from "../../api";
+
+const AddNewEvents = () => {
+  const [title, setTitle] = useState("");
+  const [weddingDecription, setWeddingDecription] = useState("");
+  const [startTime, setStartTime] = useState(new Date());
+  const [endTime, setEndTime] = useState(new Date());
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const newDateTime = new Date();
+    setStartTime(newDateTime);
+    const newDateTimeEnd = new Date();
+    setEndTime(newDateTimeEnd);
+    try {
+      const postData = await eventPostData({
+        title: title,
+        weddingDecription: weddingDecription,
+        startTime: startTime,
+        endTime: endTime,
+      });
+
+      console.log(postData);
+      debugger;
```

Comment:
remove debugger and console from code as these are the test statements and no need to push them to repository.

Reply:
removed